changes.go

```diff
@@ -472,6 +495,7 @@ type ChangeInfo struct {
 	CherryPickOfPatchSet   int                         `json:"cherry_pick_of_patch_set,omitempty"`
 	ContainsGitConflicts   bool                        `json:"contains_git_conflicts,omitempty"`
 	BaseChange             string                      `json:"base_change,omitempty"`
+	SubmitRecords          []SubmitRecord              `json:"submit_records"`
```

Comment:
Lets name the type like in the docs: `SubmitRecordInfo`

```suggestion
	SubmitRecords          []SubmitRecordInfo               `json:"submit_records,omitempty"`
```